Mathlib/Topology/Algebra/GroupCompletion.lean

```diff
@@ -115,101 +126,129 @@ instance : AddMonoid (Completion α) :=
             (continuous_map₂ (continuous_fst.comp continuous_snd)
               (continuous_snd.comp continuous_snd))))
         fun a b c ↦
-        show (a : Completion α) + b + c = a + (b + c) by repeat' rw_mod_cast [add_assoc]
-    nsmul := (· • ·)
-    nsmul_zero := fun a ↦
+        show (a : Completion α) * b * c = a * (b * c) by repeat' rw_mod_cast [mul_assoc]
+    npow := fun n ↦ Completion.map (fun a ↦ a ^ n : α → α)
```

Comment:
Losing the `:= (· • ·)` unfolding seems a bit undesirable here. Perhaps we can add a separate `Pow` instance that additivizes to `(· • ·)`?

Reply:
What do you mean?

Reply:
I pushed the change I was thinking of

Reply:
Thank you!